lib/fastlane/plugin/nowsecure/actions/nowsecure_action.rb

```diff
@@ -72,7 +74,13 @@ def self.available_options
             optional: false,
             verify_block: proc do |value|
               UI.user_error!("Could not find a file at '#{value}'") unless File.exist?(value)
-            end)
+            end),
+          FastlaneCore::ConfigItem.new(key: :nowsecure_endpoint,
+            env_name: "NOWSECURE_ENDPOINT",
+            description: "NowSecure Tenant Endpoint",
+            optional: true,
+            default_value: "https://lab-api.nowsecure.com",
+            type: String
```

Comment:
```suggestion
            type: String)
```
Found the issue. You forgot a closing paren.

For future reference, you can run check with the following:
https://github.com/WarnerMedia/fastlane-plugin-nowsecure?tab=readme-ov-file#run-tests-for-this-plugin

Reply:
sorry about that, it's fixed now.